proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -61,6 +61,12 @@ message GetLogResponse {
   SessionLog log = 1 [(buf.validate.field).required = true];
 }
 
+message DeleteRequest {
+  string session_id = 1 [(buf.validate.field).string.min_len = 1];
+}
+
+message DeleteResponse {}
+
```

Comment:
is this part of this PR ?

Reply:
explain?

Reply:
why there are changes to the sessions svc ?
you added delete to the deployments svc

Reply:
just moved to the upper part of the file. refactoring